content/industry/non-profit.mdx

```diff
@@ -0,0 +1,53 @@
+---
+heading: SSW Non-Profit Solutions
+subHeading: Customized Technology for Non-profits
+bannerImg: /images/industry/voluenteers-cover.jpg
+seo:
+  title: 'Enterprise Development and Consulting for Non-profit '
+  description: >-
+    SSW designs personalized non-profit software solutions to enhance security,
+    simplify compliance, and enrich your client relationships.
+---
+
+### Overview
+
+The non-profit sector demands flexible, cost-effective, and secure technology solutions. Data management, donor relations, fundraising optimization, and seamless user experience are some of the areas where technology can provide you with key advantages. With changing regulations and growing cyber threats, fulfilling your important responsibilities to stakeholders becomes complex. 
+
+SSW is perfectly positioned to assist. With over 30 years of experience, we design and build modern software solutions to keep non-profits at the forefront of their mission. 
+
+### How SSW can help: Customized Non-Profit Consulting and Solutions  
+
+With expertise in the non-profit sector, we create customized technology solutions that address your unique regulatory and technological challenges. Instead of providing an off-the-shelf product, we pride ourselves on building technological custom solutions to meet the distinctive needs of each non-profit organization. 
+
+We have experience with: 
+
+**Donor Management Systems:** Designing custom solutions that can be integrated with your existing software for effective management and analysis of your donor data. 
+
+**Volunteer Portals:** Creating secure and user-friendly portals to provide your volunteers with access to their tasks, tracking capabilities, and direct communication channels with you. 
+
+**Fundraising Platforms: **Building fundraising platforms that provide real-time data, intuitive interfaces, and advanced fundraising capabilities. 
```

Comment:
![image](https://github.com/SSWConsulting/SSW.Website/assets/20507092/3e6a723c-9114-465c-8950-cdc529680664)

The formatting for that bullet is broken

Reply:
fixed